### ISSUE TITLE
TLS methods should be prefixed or name-spaced with "tls" rather than suffixed 

### ISSUE THREAD

**Issue by ry** [denoland/deno#4833]
Deno.startTls -> Deno.tlsStart or Deno.tls.start
Deno.connectTls -> Deno.tlsConnect or Deno.tls.connect

**Comment by Caesar2011** [comment]
Probably better to add a TLS option to `Deno.connect` and `Deno.listen`? 🤔

Similar to what is discussed in #4769 . In this case the return type is even the same.

**Comment by ry** [comment]
listen() and listenTls() seem like very different things to me... one is a low-level open a socket - the other is invoking a new protocol server

**Comment by manyuanrong** [comment]
It is foreseeable that the content related to tls will gradually increase, reference: https://nodejs.org/api/tls.html
Therefore I recommend `Deno.tls.connect`
I'm willing to do this, it happens that I just added startTLS

**Comment by sunsetkookaburra** [comment]
I'm in favour of keeping them as is. listenTLS follows the existing left-to-right (general on left, specific on right) specificity naming convention of `read`, `readFile`, `readFileSync`. This not only makes it nicer when using code editors to find available methods (e.g. autocomplete), but I think better demonstrates the similarities in the user's view of the API. I recognise that internally `listen` and `listenTLS` may do different things, but to the user they: do perform much the same task (open a TCP socket/stream), do interact with it in a similar way and it does return the same class (`Deno.Listener`).
In short, if `listenTLS` is going to change, then why would `readSync` not become `syncRead` to follow the same naming convention; or, why should `listenTLS` change and not follow the same convention as `readSync`, etc. ?

**Comment by sunsetkookaburra** [comment]
I suppose further to this, is it preferred to group methods inside of namespaces in deno? Should Deno.connect become Deno.net.connect too?

**Comment by bartlomieju** [comment]
Done in #4888
